README.md

```diff
@@ -1,12 +1,36 @@
+# Pettamo
 
```

Comment:
A web app for online vet consultations & verified pet care services to better care for your pets.